packages/ragbits-document-search/src/ragbits/document_search/documents/document.py

```diff
@@ -32,13 +32,23 @@ class DocumentType(str, Enum):
     UNKNOWN = "unknown"
 
 
+class DocumentLocation(BaseModel):
```

Comment:
I'm wondering if that model applies for each document type that we support. For example for PDF files location may differ from MD location

Reply:
the metadata for unstructured output has document agnostic format. have inspected it and did not see anything that could be document type dependent - would leave a comment there to potentially keep in mind for the future